examples/react-app/src/index.tsx

```diff
@@ -8,6 +8,7 @@ import {
   PlmgSidebar,
   PlmgSidebarItem,
   PlmgSvgIcon,
+  PlmgSeparator,
```

Comment:
Why not show this component off in context and add it to the header component on the right slots between two menu items? 
As defined in the [header design](https://www.figma.com/file/4ZukYZAMHrsCIsrZu9u2a6/Plumage-Components?node-id=694%3A51843).